pkg/cli/cvd.go

```diff
@@ -178,3 +231,28 @@ func (o *CVDOutput) String() string {
 		fmt.Sprintf("%s/hosts/%s/cvds/%s/logs/", o.BaseURL, o.Host, o.CVD.Name)
 	return res
 }
+
+const cvdHostPackageName = "cvd-host_package.tar.gz"
+
+func listRequiredImageFilenames(listFilename, androidProductOutDir, androidHostOutDir string) ([]string, error) {
```

Comment:
nit: I think it would make more sense to pass `androidBuildTop` to this function instead of `listFilename` and construct the latter in this function.

Reply:
agree! done